hatchery/transitgateway.go

```diff
@@ -201,6 +205,19 @@ func createTransitGatewayAttachments(svc *ec2.EC2, vpcid string, tgwid string, l
 	}
 	exTg, err := svc.DescribeTransitGateways(tgInput)
 	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case "InvalidTransitGatewayID.NotFound":
+				// Accept any pending invites again
+				sess.acceptTGWShare()
+				exTg, err = svc.DescribeTransitGateways(tgInput)
```

Comment:
I'm not sure I understand the logic here: is this acquire of `exTg` redundant?

Because as the linter hinted, at https://github.com/uc-cdis/hatchery/blob/feat/multiple_paymodels/hatchery/transitgateway.go#L221 this code block returns, leaving this `exTg` value unused 🤔

Reply:
The idea is to accept any pending resource shares again, if there are none that are found. There is a race condition that happens sometimes, not sure if this is the best way to solve it tbh, but it seemed to work.